### PR TITLE
YuSa64 hse2527 #10 desktop event

### PR DESCRIPTION
**Pull request by YuSa64**
Issue #10

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: KSEA-Wisconsin | HOME
```
Comment (YuSa64) [inline events.html]: 이부분 KSEA-Wisconsin | NEWS & EVENTS 로 수정해야합니다.

File: _sass/KSEAWeb.scss [hunk 2]
```diff
@@ -373,6 +373,87 @@ footer {
       }
     }
 
+    #more_event {
+      text-align: center;
+      font-size: 30px;
+    }
+  }
+}
+.news_and_events {
```
Comment (hyecheol123) [inline _sass/KSEAWeb.scss:382]: Need to fix some settings

**Review by hyecheol123 — CHANGES_REQUESTED** [review]
Fix some variables

File: _sass/KSEAWeb.scss (hunk 2 above)
Comment (hyecheol123) [reply _sass/KSEAWeb.scss:382]: Need to fix some margin

**Review by hse2527 — APPROVED** [review]

**Pull request merged**